2024-11-19 10:34:38 - central_logger - DEBUG - WebDriverPool initialized with max_size=20
2024-11-19 10:34:38 - central_logger - DEBUG - Creating a new WebDriver instance
2024-11-19 10:34:41 - central_logger - DEBUG - WebDriver instance created and navigated to URL successfully
2024-11-19 10:34:41 - central_logger - INFO - Nom du RPA reçu: Seres
2024-11-19 10:34:41 - central_logger - INFO - Démarrage du RPA Seres avec le fichier : data\data_traitement\Rejet SERES - Correction SIRET destinataire - 20241118.xlsx
2024-11-19 10:34:41 - central_logger - INFO - Démarrage du RPA Seres avec multithreading...
2024-11-19 10:34:43 - central_logger - ERROR - Erreur lors de l'exécution du RPA Seres: 'WebDriverPool' object has no attribute 'dictionnaire_siret'
2024-11-19 10:34:43 - central_logger - INFO - Closing all WebDriver instances...
2024-11-19 10:34:46 - central_logger - INFO - All WebDriver instances closed.
& "c:/Users/PHSX095/OneDrive - LA POSTE GROUPE/Documents/GUI/.venv/Scripts/python.exe" "c:/Users/PHSX095/OneDrive - LA POSTE GROUPE/Documents/GUI/backend/server.py"
PS C:\Users\PHSX095\OneDrive - LA POSTE GROUPE\Documents\GUI> & "c:/Users/PHSX095/OneDrive - LA POSTE GROUPE/Documents/GUI/.venv/Scripts/python.exe" "c:/Users/PHSX095/OneDrive - LA 
POSTE GROUPE/Documents/GUI/backend/server.py"
 * Serving Flask app 'server'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit   
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 120-220-582
127.0.0.1 - - [19/Nov/2024 10:33:20] "POST /start-seres HTTP/1.1" 200 -
Traceback (most recent call last):
  File "C:\Users\PHSX095\OneDrive - LA POSTE GROUPE\Documents\GUI\data\rpa_modules\__init__.py", line 1, in <module>
    from .seres import SeresRPA
  File "C:\Users\PHSX095\OneDrive - LA POSTE GROUPE\Documents\GUI\data\rpa_modules\seres.py", line 571
    def start(self, excel_path="C:\Users\PHSX095\OneDrive - LA POSTE GROUPE\Documents\GUI\data\data_traitement\Rejet SERES - Correction SIRET destinataire - 20241118.xlsx"):
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape
PS C:\Users\PHSX095\OneDrive - LA POSTE GROUPE\Documents\GUI> & "c:/Users/PHSX095/OneDrive - LA POSTE GROUPE/Documents/GUI/.venv/Scripts/python.exe" "c:/Users/PHSX095/OneDrive - LA POSTE GROUPE/Document
s/GUI/backend/server.py"                                                                                                                                                                                  